src/model/messages/mrtd/EMrtdDataSubmitMessage.ts

```diff
@@ -1,9 +1,51 @@
 import { EMrtdData } from '@2060.io/credo-ts-didcomm-mrtd'
 import { Expose } from 'class-transformer'
+import * as Mrz from 'mrz'
 
 import { BaseMessage, BaseMessageOptions } from '../BaseMessage'
 import { MessageType } from '../MessageType'
 
+export type EMrtdRawData = {
+  raw: Record<string, string>
+  processed: {
+    ef_dg1: EF_DG1
+    ef_dg2: EF_DG2
+    ef_dg11?: EF_DG11
+  }
+}
+
+export type EF_DG1 = {
+  documentType: string
+  documentNumber: string | null
+  issuingState?: string | null
+  dateOfBirth?: string | null
+  dateOfExpiry?: string | null
+  sex?: string | null
+  nationality?: string | null
+  lastName?: string | null
+  firstName?: string | null
+  nameOfHolder?: string
+  mrzOptionalData?: string | null
```

Comment:
Most of these fields are mandatory, so I don't think it should be allowed them to be undefined.

BTW, this way of defining the fields will result in 3 different possibilities: undefined, string and null. I think you should choose between either returning a null when a particular, optional field (such as `mrzOptionaldata`) is not present or simply not defining it. I mean:
- `mrzOptionalData?: string` or
- `mrzOptionalData: string | null`

I'd say the first one is simpler to handle, but I don't have a strong opinion yet.

Reply:
fixed, I would prefer to leave it as `mrzOptionalData?: string`